src/Slot-Core/Behavior.extension.st

```diff
@@ -30,3 +30,14 @@ Behavior >> definesSlotNamed: aString [
 	
 	^ self slotNames includes: aString
 ]
+
+{ #category : #'*Slot-Core' }
+Behavior >> hasSpecificLayout [
+	"Return true whether the receiver has a layout that is not a fixed one (which is the default linear layout for default classes."
+	"Point hasSpecificLayout >>> false"
+	"Array hasSpecificLayout >>> true"
+	
+	^ (self classLayout isKindOf: FixedLayout) not
```

Comment:
Can we have AbstractLayout>>#isSpecific (or isFixed) to avoid isKindOf:?

Reply:
This is a good suggestion. Can you open an issue after this one is integrated.
right now I want to use this method since this code is used in the slot definition (so the kind of is already there).